contracts/src/CrossChainApplications/ERC20Bridge/tests/ERC20BridgeTests.t.sol

```diff
@@ -81,9 +81,7 @@ contract ERC20BridgeTest is Test {
 
         vm.expectCall(
             MOCK_TELEPORTER_REGISTRY_ADDRESS,
-            abi.encodeWithSelector(
-                WarpProtocolRegistry.getLatestVersion.selector
-            )
+            abi.encodeWithSignature("latestVersion()")
```

Comment:
Doesn't matter to me either way, but just curious if it's possible to use `TeleporterRegistry.latestVersion.selector` version here or not?

Reply:
updated, for the public state variable getter functions, the access to their function is different, needed to use an isntance of the contract